submissions/factorio_draftsman/main.md

```diff
@@ -0,0 +1,211 @@
+
+* **Draftsman does it all.** All types of entities are supported, from splitters to stack inserters. If you can do it in game, you can do it with Draftsman; allowing you to focus on the one problem you're actually trying to solve.
+* **Draftsman is uni-language.** Written in Python, this makes it exceptionally easy to install, simple to use, and gives the user access to the entirety of Python's vast packaging index. Chances are, you can do it in Python with Draftsman, regardless of *what* Factorio-related thing you're actually doing.
+* **Draftsman is easy to use.** Designed from the start to be simple and (most importantly) self-documenting, Draftsman allows you to manipulate blueprints and entities by auto-completable attributes and methods.
+* **Draftsman is well-documented.** Every function, method, attribute, and class is documented and crosslinked at it's [readthedocs site](https://factorio-draftsman.readthedocs.io/en/latest/?badge=latest). In addition, tutorials and supplementary material is provided, as well as a whole host of [example programs](https://github.com/redruin1/factorio-draftsman/tree/main/examples) to help illustrate how Draftsman works.
+* **Draftsman is stable.** A rigorous suite of tests ensure that Draftsman behaves predictably and correctly (or at least correctly enough), with a code coverage of 100%. Draftsman is verified to work on the latest versions of Python 2 and 3, and is compatible with all versions of Factorio greater than 1.0, with the metrics to prove it.
+* **Draftsman is descriptive.** Draftsman enforces "Factorio-safety" as a core philosophy, which means that if there is any change to a blueprint that would result in an error on import, an exception is thrown. Draftsman also tries to enforce "Factorio-correctness", which means that values that won't break, but are otherwise nonsensical, will raise warnings. Both errors and warnings are verbose, so that any problem with your script can be understood and resolved in seconds.
+* **Draftsman is close-to-source.** Draftsman bases all of its data off of [Wube's `factorio-data` repository](https://github.com/wube/factorio-data), which means that all entities are exactly as you would expect in-game with no inconsistencies. This makes Draftsman up-to-date, makes updating Draftsman for future Factorio versions a breeze, and allows versioning control to monitor changes between Draftsman and Factorio if any future breakage should occur.
+* **Draftsman supports mods.** Draftsman emulates [Factorio's data lifecycle](https://lua-api.factorio.com/latest/Data-Lifecycle.html) directly, meaning that the same loading process that happens when you launch the game is mimicked with a single Draftsman function. In addition to ensuring absolute accuracy to Factorio, this also makes it compatible with custom mod prototypes that can be accessed in script as if they were any other vanilla entity.
```

Comment:
> that can be accessed in script

What do you mean by this?

Reply:
Draftsman that uses Python. This is just saying that modded entities are stored in the same place in Draftsman as vanilla ones; if you update your installation with a set of mods and then print `draftsman.data.entities.raw` it will display all the entities currently loaded, vanilla or modded. This mimics how it works in Factorio, where a modded entity is just added to the list of prototypes of that entity's type; a modded accumulator would just get added to the list of all accumulators, from which you can query via Draftsman script.